include/dnnl.hpp

```diff
@@ -69,7 +101,7 @@ struct error : public std::exception {
     /// @param status The error status returned by the C API.
     /// @param message The error message.
     static void wrap_c_api(dnnl_status_t status, const char *message) {
-        if (status != dnnl_success) throw error(status, message);
+        if (status != dnnl_success) DNNL_THROW(status, message); 
```

Comment:
I've counted 8 cases when we `throw` in this file at latest master, not 4 as in this patch. Is it desired behavior we leave some of them untouched?

Reply:
done